app/models/candidate.rb

```diff
@@ -2,6 +2,8 @@ class Candidate < ApplicationRecord
   include Chased
   include AuthenticatedUsingMagicLinks
 
+  generates_token_for :unsubscribe_link
```

Comment:
I love that this "just works"™️

Reply:
Yeah, I am thinking about all the code I wrote in a rails 5/6 app a few years ago to do this. And here it is, just one line...